frontend/src/components/pages/EditAccountPage.tsx

```diff
@@ -163,7 +163,7 @@ const EditAccountPage = (): React.ReactElement => {
       {isError && <ErrorModal />}
       <Container maxW="container.xl" align="left" mt={12}>
         <Text mb={2} textStyle="display-large">
-          Account Creation
+          Edit Account
```

Comment:
```suggestion
          Edit Profile
```
Actually this naming could be a bit better?